share/completions/signify.fish

```diff
@@ -0,0 +1,4 @@
+complete -c signify -n '__fish_use_subcommand' -s C --description 'Verify a signed checksum list'
```

Comment:
`__fish_use_subcommand` ignores things that start with a dash, so this would have no effect.

Reply:
Use `__fish_seen_subcommand_from`.